### PR TITLE
translations: Research translations for Indonesian.

### PR DESCRIPTION
**Pull request by rafidaslam**

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@rafidaslam, it looks like you've already signed the Dropbox CLA. Thanks!

**Comment by trueskawka** [comment]
Hi @rafidaslam!

Thanks for working on this!

I don't know Indonesian, but we have a mentor who's fluent in the language. Before I ask him for a review, could you add English translations to your docs file? Would be easier to follow for those who can't speak the language :)

Thanks!

**Comment by rafidaslam** [comment]
@trueskawka Thanks, for looking at this! Updated

**Comment by trueskawka** [comment]
Thanks for updating!

@vaibhavsagar could you take a look?

File: translations/Indonesian/research/notes.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+|:---------------------:|:-----------------------------:|:------------------------:|
+| Messages              | Pesan                         | Pesan-pesan              |
+| You                   | Anda                          | Kamu                     |
+| I                     | Saya                          | Aku                      |
+| Home                  | Beranda                       | Rumah                    |
+| View all              | Lihat semua                   | Tampakkan semua          |
+| View all              | Tampilkan semua               | Tampakkan semua          |
+| Delete                | Hapus                         | Buang                    |
+| Add                   | Tambahkan                     | Menambahkan              |
+| Change                | Ubah                          | Ganti                    |
```
Comment (vaibhavsagar) [inline translations/Indonesian/research/notes.md]: 'Berubah' instead of 'Ubah'?

Comment (rafidaslam) [inline translations/Indonesian/research/notes.md]: "Berubah" is more like "Changed" in english

Comment (vaibhavsagar) [inline translations/Indonesian/research/notes.md]: Betul sekali! Carry on :smile:

**Comment by vaibhavsagar** [comment]
This looks good to me.

**Comment by rafidaslam** [comment]
Thanks for the review @vaibhavsagar !

**Comment by hackerkid** [comment]
This looks awesome. Merged and approved the task. :tada:

**Pull request merged**